libautoscoper/src/Tracker.cpp

```diff
@@ -579,21 +579,73 @@ std::vector <double> Tracker::trackFrame(unsigned int volumeID, double* xyzypr)
       double viewport[4];
       this->calculate_viewport(modelview, viewport);
 
+      // RAD SPAn     // Export Full Images
+      double rad_viewport[4];
+      rad_viewport[0] = -views_[i]->camera()->viewport()[2] / 2;
+      rad_viewport[1] = -views_[i]->camera()->viewport()[3] / 2;
+      rad_viewport[2] = views_[i]->camera()->viewport()[2];
+      rad_viewport[3] = views_[i]->camera()->viewport()[3];
```

Comment:
This is something that I am having trouble wrapping my head around. From my understanding, the camera viewport is defined as [x_min, y_min, width, height]. So why are you not using `views_[i]->camera()->viewport()[0]` and `views_[i]->camera()->viewport()[1]` for `rad_viewport[0]` and `rad_viewport[1]`?

https://github.com/BrownBiomechanics/Autoscoper/blob/4bdedd802034a1c341099d735010690105a106ad/libautoscoper/src/Camera.cpp#L438-L449

https://github.com/BrownBiomechanics/Autoscoper/blob/4bdedd802034a1c341099d735010690105a106ad/libautoscoper/src/Tracker.cpp#L290-L293

https://github.com/BrownBiomechanics/Autoscoper/blob/4bdedd802034a1c341099d735010690105a106ad/libautoscoper/src/gpu/opencl/RadRenderer.cpp#L102-L108